src/encoded/static/components/search.js

```diff
@@ -1043,9 +1042,9 @@ const SeriesComponent = ({ context: result, cartControls, removeConfirmation, au
                             </span>
                         : null}
                     </a>
-                    {fccSeries && (result.description || result.biosample_summary) ?
+                    {fccSeries ?
```

Comment:
Do you not need `(result.description || result.biosample_summary)`? You might not; it depends whether the empty `<div className="result-item__highlight-row">` has an impact or not.